### ISSUE TITLE
Feature request: ExclusionImportOption supports package excludes 

### ISSUE THREAD

**Issue by KayWeinert** [societe-generale/arch-unit-build-plugin-core#46]
## Type of Issue

It is a : 
 - [ ] bug
 - [x] request
 - [ ] question regarding the documentation

## Motivation
I have implemented a solution for the "exclude generated-sources feature request" (https://github.com/societe-generale/arch-unit-maven-plugin/issues/33) but there is a little problem with ExclusionImportOption#includes method.
The include check is based on a file path. My solution generates a package path as exclude pattern and so ExclusionImportOption#includes always returns true.

My request is it to extend the include check with some kind of package check.

## Current Behavior

~~~
 @Override
    public boolean includes(Location location) {

        if(location.contains(patternToExclude)){
            return false;
        }
        return true;
    }
~~~

## Expected Behavior

~~~
 @Override
    public boolean includes(Location location) {

       if(location.contains(patternToExclude) || location.contains(patternToExclude.replaceAll("\\Q.\\E", File.separator)){
            return false;
        }
        return true;
    }

~~~

**Comment by vincent-fuchs** [comment]
hi, thanks for your suggestion. Can you please submit a pull request for this, with the relevant tests to cover for this new behavior ? 

thanks

**Comment by vincent-fuchs** [comment]
Thanks for your contirbution ! change is available in v2.7.3 - I'll release the maven and gradle plugins shortly
